App/RequestList.js

```diff
@@ -98,16 +98,19 @@ class RequestList extends React.Component {
 
 
   getAvailableRequests(index){
-    var selectedIndex = index;
+
     return(
       <View style={styles.spacer}>
-      <TouchableHighlight onPress={this.props.goToVenueRequests.bind(this, selectedIndex)}>
+      <TouchableHighlight onPress={this.props.goToVenueRequests.bind(this)}>
```

Comment:
the deleted line was correct. But it should be, `this.props.goToVenueRequests.bind(this, index)` as selectedIndex is not defined